ansible_base/rbac/api/serializers.py

```diff
@@ -180,7 +180,7 @@ def get_fields(self):
         return fields
 
     def raise_id_fields_error(self, field1, field2):
-        msg = _(f'Provide exactly one of {self.actor_field} or {self.actor_field}_ansible_id')
+        msg = _('Provide exactly one of %(actor_field)s or %(actor_fields)s_ansible_id') % {'actor_field': self.actor_field}
```

Comment:
I don't know how I let such a rookie mistake in!

Reply:
`%(actor_fields)` --> `%(actor_field)`

Reply:
good catch, thank you good sir.